crates/block-producer/src/mempool/mod.rs

```diff
@@ -366,11 +431,11 @@ mod tests {
         }
     }
 
-    // BATCH REVERSION TESTS
+    // BATCH FAILED TESTS
     // ================================================================================================
```

Comment:
nit: should we move the tests into a separate file? (e.g., `src/mempool/tests.rs`)